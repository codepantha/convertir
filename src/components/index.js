export { default as Card } from './card/Card';
export { default as Header } from './header/Header';
export { default as Navigation } from './navigation/Navigation';
export { default as Filter } from './filter/Filter';
export { default as Loader } from './loader/Loader';
