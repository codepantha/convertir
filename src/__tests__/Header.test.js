import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import { Header } from '../components';

test('it renders correctly', () => {
  const tree = renderer.create(<Provider store={store}><Header /></Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});
