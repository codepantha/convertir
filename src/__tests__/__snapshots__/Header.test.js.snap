// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="convertir__header"
  data-testid="heading"
>
  <h1>
    
  </h1>
</div>
`;
